docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1165,6 +1165,11 @@ to the connector using a :doc:`WITH </sql/create-table-as>` clause::
       ``s3a://test/name=${name}/``. Mapped from the AWS Athena table property
       `storage.location.template <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-setting-up.html#partition-projection-specifying-custom-s3-storage-locations>`_
     - ``${table_location}/${partition_name}``
+  * - ``extra_properties``
+    - Additional properties added to a Hive table. The properties are not used by Trino,
+      and are currently not shown in SHOW CREATE TABLE statements. However, these properties
```

Comment:
How about 

```
Additional properties added to a Hive table. The properties are not used by Trino, and are available in the ``$properties`` metadata table. The properties are not included in the output of ``SHOW CREATE TABLE`` statements.
```

Reply:
Also note.. I used the term metadata table .. like in the Iceberg connector doc .. since I believe thats the better terms.. system table is more like from the system catalog .. with Trino as the system. Does that make sense? Also .. we will really have to add a separate section about that metadata table after this PR is merged. And maybe others

Reply:
Should we mention `The properties are not included in the output of` it could changed in the future ?

Reply:
@mosabua AC